_posts/2021-11-19-connecting-java-high-level-rest-client-with-opensearch-over-https.markdown

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "Connecting to OpenSearch over HTTPS using the Java High-Level REST Client"
+authors: 
+  - setiah
+date: 2021-11-19 01:01:01 -0700
+categories: 
+  - technical-post
+twittercard:
+  description: "This post walks through the steps to configure Java High-level REST client to connect to OpenSearch over HTTPS."
+---
+
+If you have ever used OpenSearch with a Java application, you might have come across OpenSearch Java high-level REST client. The REST client provides OpenSearch APIs as methods, and makes it easier for a Java application to interact with OpenSearch using request/response objects.
+
+In this blog, we will see how to configure the Java high-level REST client to connect to an OpenSearch cluster over HTTPS. We will set up a local OpenSearch server with the default distribution, and use the client in a sample Java application to connect to the server. I have used an Ubuntu 18.04 machine, but it should work for other linux distributions as well.
+
+**Setup OpenSearch Server**
+
+Lets start with downloading the [latest OpenSearch linux distribution](https://opensearch.org/downloads.html) from [opensearch.org](http://opensearch.org/). I have used 1.1.0 version which is the latest available version at the time of this writing. Unzip the downloaded tar.gz file.
+
+```
+> wget https://artifacts.opensearch.org/releases/bundle/opensearch/1.1.0/opensearch-1.1.0-linux-x64.tar.gz
+
+> tar -xf opensearch-1.1.0-linux-x64.tar.gz
```

Comment:
About half way through the blog you stopped starting each command line with `>` - make it consistent.